centinel/primitives/traceroute.py

```diff
@@ -119,20 +120,18 @@ def traceroute(domain, method="udp", cmd_arguments=None,
 
     hops = list()
     for hop in parsed_output.hops:
-        hop_json = { "index" : hop.idx, "asn" : hop.asn }
+        hop_json = {"index": hop.idx, "asn": hop.asn}
         probes_json = []
         for probe in hop.probes:
-            probes_json.append({ "name" : probe.name,
-                              "ip" : probe.ip,
-                              "rtt" : probe.rtt,
-                              "anno" : probe.anno })
+            probes_json.append({"name": probe.name,
+                                "ip": probe.ip,
+                                "rtt": probe.rtt,
+                                "anno": probe.anno})
         hop_json["probes"] = probes_json
         hops.append(hop_json)
 
-    results = {}
     results["dest_name"] = parsed_output.dest_name
     results["dest_ip"] = parsed_output.dest_ip
-    results["method"] = method
```

Comment:
Cause I found that in all circumstances "method" is always in results dict, so I removed redundant results["method"] = method and moved them to line 45 of the same file.